Doc/whatsnew/3.14.rst

```diff
@@ -2916,154 +2919,152 @@ New features
   * :c:func:`PyLongWriter_Finish`
   * :c:func:`PyLongWriter_Discard`
 
-  (Contributed by Sergey B Kirpichev and Victor Stinner in :gh:`102471`.)
+  (Contributed by Sergey B. Kirpichev and Victor Stinner in :gh:`102471`.)
```

Comment:
Let's stick with Sergey's own chosen spelling, which is already used a lot elsewhere.
```suggestion
  (Contributed by Sergey B Kirpichev and Victor Stinner in :gh:`102471`.)
```
(Here and elsewhere in this PR)